CONTRIBUTING.md

```diff
@@ -222,3 +222,15 @@ $ apt install lcov
 $ lcov --capture --directory . --output-file coverage.info
 $ genhtml -o html_report coverage.info
 ```
+
+## Debugging
+
+### Checking the UMF version and CMake variables
+
+Strings with the UMF version and useful CMake variables can be grepped in the following way:
+
+```bash
+$ strings libumf.so | grep "@(#)"
```

Comment:
`strings` is part of `binutils` package on Ubuntu (not sure if we want to mention it here..., as a requirement)...?